### DISCUSSION TITLE
OpenID Set as default and failing so I can't login.

### DISCUSSION THREAD

**Discussion by nick-johannsen** [OrchardCMS/OrchardCore#16610] [Q&A]
I have enable Orchard Core OpenID and got it mostly working. When troubleshooting some registration issues I accidentally set it to only use the external login which is not fully working. Now I am unable to log into my admin panel. 

Is there a way to get to local login when OpenID is set to login in through external only?
Can I disable the OpenID module out side the web?
Can I change configuration on OpenID Module out side the web?
Any other Ideas to get access back?

**Comment by MichaelPetrinolis** [comment]
You can open the document table and search for the settings document. Format the document in an editor, change the local login setting, minify the document and update the record. You should restart the app and you can login with the local password.

**Comment by onestar1** [comment]
how to do？
